v-next/hardhat-utils/src/sub-process.ts

```diff
@@ -0,0 +1,27 @@
+import { spawn } from "node:child_process";
+
+/**
+ * Spawns a detached subprocess to execute a given file with optional arguments.
+ *
+ * @param absolutePathToSubProcessFile - The absolute path to the subprocess file to be executed.
+ * @param [args=[]] - Optional arguments to pass to the subprocess.
+ *
+ * The subprocess runs in a detached mode and its standard input/output is ignored.
+ * This function does not wait for the subprocess to complete and the subprocess is unreferenced
+ * to allow the parent process to exit independently.
+ */
+export function spawnDetachedSubProcess(
+  absolutePathToSubProcessFile: string,
+  args: string[] = [],
+): void {
+  const subprocess = spawn(
+    process.execPath,
+    ["--import", "tsx/esm", absolutePathToSubProcessFile, ...args],
+    {
+      detached: true,
+      stdio: "ignore",
```

Comment:
I'm not sure if this works on windows, see `https://nomic-labs.sentry.io/issues/5567768827/?referrer=slack&notification_uuid=5783a85c-f818-4109-be8a-5de9dfdf3212&alert_rule_id=1079337&alert_type=issue`

But we can address it later.

Reply:
Can you add a `console.log` and `console.error` call to your test to make sure they still work?